deluge/ui/gtkui/glade/add_torrent_dialog.ui

```diff
@@ -56,15 +102,14 @@
                     <property name="left_padding">12</property>
                     <property name="right_padding">12</property>
                     <child>
-                      <object class="GtkVBox" id="vbox2">
+                      <object class="GtkBox" id="vbox2">
                         <property name="visible">True</property>
                         <property name="can_focus">False</property>
+                        <property name="orientation">vertical</property>
                         <child>
                           <object class="GtkScrolledWindow" id="scrolledwindow1">
                             <property name="visible">True</property>
                             <property name="can_focus">True</property>
-                            <property name="hscrollbar_policy">automatic</property>
-                            <property name="vscrollbar_policy">automatic</property>
                             <property name="shadow_type">in</property>
                             <child>
                               <object class="GtkTreeView" id="listview_torrents">
```

Comment:
To make it clearer what is being referenced in code, should rename `listview_torrents` to `treeview_torrents` and obviously change all references in  `deluge/ui/gtkui/addtorrentdialog.py`